packages/flutter_tools/lib/src/macos/cocoapods.dart

```diff
@@ -371,7 +371,7 @@ class CocoaPods {
       ).send();
       _logger.printError(
         'Error: To set up CocoaPods for ARM macOS, run:\n'
-        '  arch -x86_64 sudo gem install ffi\n',
+        '  sudo gem uninstall ffi && sudo gem install ffi -- --enable-libffi-alloc\n',
```

Comment:
What does the additional `--` between "ffi" and "--enable-libffi-alloc" mean?

Reply:
I guess they aren't parsing it as a flag but as a string positional parameter? I don't know why they did it like that, but the `--` is needed.
https://github.com/ffi/ffi/issues/800#issuecomment-1025309396